common/types/explorer.ts

```diff
@@ -424,11 +424,11 @@ export interface GridSortingColumn {
 }
 
 export enum DirectQueryLoadingStatus {
-  SUCCESS = 'SUCCESS',
-  FAILED = 'FAILED',
-  RUNNING = 'RUNNING',
-  SCHEDULED = 'SCHEDULED',
-  CANCELED = 'CANCELED',
+  SUCCESS = 'success',
```

Comment:
Should we add `waiting` here. Is that a valid status?

Reply:
We could add it, but it currently wouldn't be used for anything since the raw status is displayed to the UI.